app/views/transactionsAddReqResModal.html

```diff
@@ -1,127 +1,180 @@
+<style>
```

Comment:
Ok, it not too important for now, but ideally if we find a better way it would be better.